pulpcore/cli/common/generic.py

```diff
@@ -30,6 +30,15 @@
 
 
 class PulpCommand(click.Command):
+    def __init__(
+        self,
+        *args: Any,
+        needs_contexts: Optional[Tuple[Type[PulpEntityContext]]] = None,
+        **kwargs: Any,
+    ):
+        self.needs_contexts = needs_contexts if needs_contexts else None
```

Comment:
```suggestion
        self.needs_contexts = needs_contexts
```
In the worst case this could be an empty tuple. And if you think we _need_ to normalize, it would be `needs_contexts or None`.